### ISSUE TITLE
Fatal error after composer dumpautoload

### ISSUE THREAD

**Issue by mediaessenz** [helhum/dotenv-connector#26]
I use this package inside a TYPO3 10.4 installation running under PHP 7.4

Before installing it, I had no issue with `composer dumpautoload`, but after I get this error message:
```
PHP Fatal error:  Uncaught Error: Call to undefined function Symfony\Component\Dotenv\trigger_deprecation() in /Users/alex/Projects/domain.tld/app/vendor/symfony/dotenv/Dotenv.php:49
Stack trace:
#0 /Users/alex/Projects/domain.tld/app/vendor/helhum/dotenv-include.php(4): Symfony\Component\Dotenv\Dotenv->__construct('1')
#1 /Users/alex/Projects/domain.tld/app/vendor/helhum/dotenv-include.php(6): Helhum\DotEnvConnector\IncludeFile::{closure}('/Users/alex/Pro...')
#2 /Users/alex/Projects/domain.tld/app/vendor/helhum/dotenv-connector/src/IncludeFile.php(62): require('/Users/alex/Pro...')
#3 /Users/alex/Projects/domain.tld/app/vendor/helhum/dotenv-connector/src/Plugin.php(99): Helhum\DotEnvConnector\IncludeFile->dump()
#4 [internal function]: Helhum\DotEnvConnector\Plugin->onPreAutoloadDump(Object(Composer\Script\Event))
#5 phar:///usr/local/Cellar/composer/1.10.10/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php(164): call_user_func(Array, Object(Composer\Script\Event))
#6 phar:///usr/local/Cellar/ in /Users/alex/Projects/domain.tld/app/vendor/symfony/dotenv/Dotenv.php on line 49
```

To get this command working again, I'm currently commented the trigger_deprecation call, but this is maybe not the best idea :-)

Interesting is, that my PhpStorm can find the trigger_deprecation definition inside the symfony framework.

I already deleted my composer.lock and the whole vendor dir and did a `composer install` without any effect.

**Comment by helhum** [comment]
what version of symfony/dotenv do you have installed? and which version of helhum/dotenv-connector?

**Comment by helhum** [comment]
you need the latest version of helhum/dotenv-connector, which currently still conflicts with symfony/dotenv >= 5.1.0

**Comment by mediaessenz** [comment]
I use version 2.3.1 helhum/dotenv-connector and version 5.1.0 of symfony/dotenv

**Comment by helhum** [comment]
> I use version 2.3.1 helhum/dotenv-connector and version 5.1.0 of symfony/dotenv.

Are you sure?

If so: do you know how you got these packages installed together in these versions? It should be impossible, as 2.3.1 of dotenv-connector conflicts all versions of symfony/dotenv higher or equal 5.1.0

In my tests I cannot get Composer to install these two versions alongside each other in one project

**Comment by helhum** [comment]
> What should I do to get it working?

Basically downgrade symfony/dotenv to a version lower than 5.1.0

**Comment by mediaessenz** [comment]
Add "symfony/dotenv": "5.0.11" as a reqirement to my composer.json and did a composer update. 
composer dumpautoload works again :-)

**Comment by helhum** [comment]
@mediaessenz I would still be interested how you could install these versions together. Would be great if you could provide feedback on that. Thanks

**Comment by helhum** [comment]
Fixed with a18cede and released as v3.0.0

**Comment by mediaessenz** [comment]
It could be, that I did a ‚composer require symfony/dotenv‘, before I finally required this package.
I don’t know why this worked at all, because of the conflict section, but that‘s how it was.
